src/fragments/start/getting-started/react/auth.mdx

```diff
@@ -58,6 +59,6 @@ You can also customize this component to add or remove fields, update styling, o
 
 In addition to the `withAuthenticator` you can build custom authentication flows using the `Auth` class.
 
-`Auth` has over 30 methods including `signUp`, `signIn`, `forgotPassword`, and `signOut` that allow you full control over all aspects of the user authentication flow. Check out the complete API [here](https://aws-amplify.github.io/amplify-js/api/classes/authclass.html)
+`Auth` has over 30 methods including `signUp`, `signIn`, `forgotPassword`, and `signOut` that allow you full control over all aspects of the user authentication flow. Check out the complete API and docs [here](https://ui.docs.amplify.aws/components/authenticator)
```

Comment:
Depending on the answer to the above, this section made need to be removed entirely

Reply:
Yep, even if we keep it, this should still link to the JS API docs that was there previously.

IMO, since this is a comprehensive intro to Amplify JS *and* UI, we can keep it here for those interested in more fine-tuned Auth.